lib/tasks/historic_data_contacts_whitespace_strip.rake

```diff
@@ -0,0 +1,13 @@
+namespace :strip_whitespace do
+  desc 'Strip whitespace from specified historic Contact entrys (one off task)'
+  task data_cleanup: :environment do
+    BATCH_SIZE = 1000
+    # Process in batches set by the batch size 1000 is default.
+    Contact.find_in_batches(batch_size: BATCH_SIZE) do |contact|
+      puts contact.size
```

Comment:
If we have 5 100 records we will have:
1000
1000
1000
1000
100

We can just have some ids output like:
12,13,14,15....
to show the progress

Reply:
There was a progress meter in the previous one. I just made this super simple. Maybe too simple. @go8soft what about failures? There should be none, but how do you propose to display those?

This was why in the closed PR I wrote a sql statement to execute so we knew that only records that needed updating are targeted. It should in theory be a small subset of the total number of records (hopefully)

Reply:
sa`about failures` thanks a good point, thanks! Can we do `save!` instead of `save`?

Reply:
@go8soft Actioned as requested